### PR TITLE
Front End Client + Relay Setup

### PR DESCRIPTION
**Pull request by sam-warren**

### CODE REVIEW

File: schema/deploy/todo_app.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+-- Deploy todo:todo_app to pg
+
+BEGIN;
+
+-- XXX Add DDLs here.
+CREATE SCHEMA if not exists todo_app;
+
+COMMIT;
```
Comment (pbastia) [inline schema/deploy/todo_app.sql:8]: The team's coding style is to have all SQL in lowercase

File: schema/deploy/todo_data.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
+-- Deploy todo:todo_data to pg
+-- requires: todo_app
+
+BEGIN;
+
+-- XXX Add DDLs here.
+create table if not exists todo_data (
```
Comment (pbastia) [inline schema/deploy/todo_data.sql:7]: the table needs to be created on the `todo_app` schema
```suggestion
create table if not exists todo_app.todo_data (
```

Comment (sam-warren) [reply schema/deploy/todo_data.sql]: Good catch!

File: schema/deploy/todo_app.sql (hunk 1 above)
Comment (sam-warren) [reply schema/deploy/todo_app.sql]: Will update this.

**Pull request closed without merging**